tests/test_DataAugmenterInternally.py

```diff
@@ -0,0 +1,156 @@
+import os
+import sys
+import pytest
+import numpy as np
+import pandas as pd
+
+from scipy.stats import ks_2samp
+
+sys.path.append("zarnitsa/")
+
+from DataAugmenterInternally import DataAugmenterInternally
+
+"""
+Under the null hypothesis the two distributions are identical.
+If the K-S statistic is small or the p-value is high
+(greater than the significance level, say 5%),
+then we cannot reject the hypothesis that the distributions of
+the two samples are the same.
+Conversely, we can reject the null hypothesis if the p-value is low.
+"""
+
+N_TO_CHECK = 500
+
+
+@pytest.fixture
```

Comment:
if you use absolutely the same fixtures in two different files inside one directory, consider moving it to `conftest.py`, it will be available from it automatically (you even haven't to import it inside test files)

Reply:
sorry, it was a mistake there. I accidentally did it